src/components/visualizations/chart/test/chartOptionsBuilder.spec.ts

```diff
@@ -291,6 +293,36 @@ describe('chartOptionsBuilder', () => {
                 });
             });
         });
+
+        describe('optional stacking with viewBy 2 attributes', () => {
+            const chartOptions: IChartOptions = {
+                isViewByTwoAttributes: true,
+                type: 'column'
+            };
+
+            it.each([
```

Comment:
In this case, I would just make two test cases and remove `it.each`. The readability will be better imho.

Reply:
I think set data of less than 4 for 5 is still readable :)

Reply:
I meant that the `it.each` should be the last resort and separated test cases can be understood better than putting it all together from two different places. However, I don't fixate on it as this can be individual :)

Reply:
@naibinh hello, we were just talking about this pattern on JS Sync meeting. You will see meeting notes and updated pull request checklist soon in #javascript. We will discourage to use the test parametrization in cases when whole "should..." text is in the `each section. Each test case should have a string name and not just `%s` as it makes tests less readable. You should only replace input and output values in the test case name template and rather use utility methods to generate data for tests than `it.each`.

Reply:
Hi @xMort ,
I have refactored it :)
Please take a look. Thanks

Reply:
It's better. You can see how is this pattern not ideal in this case because you have two underscored parameters that are not used in the test, only in its name. Ideally, you would want to do parametrized test only for input/output array and use these both in test name and body.

Please, resolve conflicts, squash commits and lets merge it.